templates/volunteers/profile.html

```diff
@@ -237,4 +237,42 @@ <h2>
         </div>
     </div>
 </div>
+
+<div class="container">
```

Comment:
You don't need to create a new container element here. Just use the previous container and start with a new row.